src/components/OSCALControlProse.js

```diff
@@ -64,21 +64,13 @@ function getParameterValue(
   implReqSetParameters,
   parameterId
 ) {
-  // Locate matching parameter to parameterId
-  let foundParameterSetting = null;
-  // Locate set-parameters when found in the by-component
-  if (statementByComponent?.["set-parameters"]) {
-    foundParameterSetting = statementByComponent["set-parameters"].find(
-      (parameterSetting) => parameterSetting["param-id"] === parameterId
-    );
-    // Locate set-parameters in the control's initial implemented requirement statement
-  } else if (implReqSetParameters) {
-    foundParameterSetting = implReqSetParameters.find(
-      (parameterSetting) => parameterSetting["param-id"] === parameterId
-    );
-  } else {
-    return null;
+  function parameterHasGivenId(parameterSetting) {
+    return parameterSetting["param-id"] === parameterId;
   }
+  // Locate set-parameters when found in the by-component
+  const setParameters = () =>
+    statementByComponent?.["set-parameters"] || implReqSetParameters;
+  const foundParameterSetting = setParameters()?.find(parameterHasGivenId);
```

Comment:
```suggestion
  // Locate set-parameters when found in the by-component
  const setParameters = statementByComponent?.["set-parameters"] || implReqSetParameters;
  const foundParameterSetting = setParameters?.find(parameterHasGivenId);
```

We actually don't need to make this a function since we never use it again. Previously it was a little more necessary but with the rest of the cleanup, we don't need it anymore